src/olympia/landfill/collection.py

```diff
@@ -33,18 +34,17 @@ def create_collection(application):
     return c
 
 
-def generate_collection(addon, app=None):
+def generate_collection(addon, app=None, **kwargs):
```

Comment:
@eviljeff I modified a few of the generate commands to pass an app to the ```generate_collection``` script. This allowed me to tie a generated collection to a specific user.